### ISSUE TITLE
Design of a brand repository

### ISSUE THREAD

**Issue by PhilippePlagnol** [okfn/product-browser-web#2]
In order to store a product into the database, a brand must be assigned.

The current data about brand and groups (id, name, picture, website, like to wikipedia articles,...) has been created manually).

A collaborative platform to manage this repository is critical for the project.

There's an important constraint : the brand identifier must be stable,...

**Comment by rufuspollock** [comment]
@PhilippePlagnol can you give a bit more info about what you mean by the brand for a product - perhaps a concrete example ...

**Comment by PhilippePlagnol** [comment]
Yes, no prob : in the database, there're currently 3 tables for this brand repository : brand, brand_type and brand_group   

just go here for the example : http://www.product-open-data.com/group-list/
The list that you see is the "groups" stored in the brand_group table

if you click on "Pepsico", you see that we are speaking of the the corporate group "Pespico" owning a lot of brands (including of course "Pepsi".

For the group, we have the logo, the website and a link to wikipedia for more details

now, click on the brand called "Izze"
We have a logo, a website and a type (manufacturer brand, retailer brand ,... - this labels are stored in the brand_type table) 

All the product you see are the product with the brand_cd = 3629 which is the ID of the brand "Izze" in our repository.

So when you add a product in the database, you need to assign a brand and this is done manually because there's no public and worldwide brand repository for the moment,... we are building it ;)

**Comment by rufuspollock** [comment]
So strictly "brand" is what you have in `brand_type` table and some brands are part of a Group (i.e. an owning entity) which are listed in `brand_group` table.

Have I understood correctly?

**Comment by PhilippePlagnol** [comment]
you can see the description of tables in this document: http://www.product-open-data.com/docs/POD-SPECS-2013.10.08_01.xlsx

brands are stored in the brand table, groups are stored in the group table, brand_type only contains 3 records
1 => Manufacturer-brand
2 => Retailer-brand
3 => The GLN must be checked before to assign a brand 

brand_type_cd is a foreign key in the brand table

example : pepsi is a manufacturer brand, and the Tesco brands (http://realfood.tesco.com/our-food/our-brands.html) are retailer brands,...

Have I answered to your question correctly ?
